build-system/tasks/process-github-issues.js

```diff
@@ -102,103 +103,111 @@ function getIssues(opt_page) {
  * depending if missing milestone or label,
  * tasks applied as per design go/ampgithubautomation
  */
-function updateGitHubIssues(){
-  var promise = Promise.resolve();
+function updateGitHubIssues() {
+  let promise = Promise.resolve();
   return BBPromise.all([
     getIssues(1),
     getIssues(2),
     getIssues(3),
     getIssues(4),
     getIssues(5),
-    ])
-  .then(requests => [].concat.apply([], requests))
-  .then(issues => {
-    const allIssues = issues;
-    allIssues.forEach(function(issue) {
-      var labels = issue.labels;
-      var issueType;
-      var milestone = issue.milestone;
-      var milestoneTitle;
-      var milestoneState;
-      var hasPriority = false;
-      var issueNewMilestone = MILESTONE_PENDING_TRIAGE;
+  ])
+      .then(requests => [].concat.apply([], requests))
```

Comment:
The lint fixes here appear to have messed up the indentation. It's possible that the linter rules used were for a different version. Also, these fixes will fail on Travis because we use node 4 on Travis for our build tools.

Please revert the automated indentation fixes and stick with the comments from the last review. There's an easy way to revert just that commit and push to the branch once again.